frontend.py

```diff
@@ -7,514 +8,493 @@
-				printColor("return key: start a new line.",getConfig(2),"\n")
-				newContent1 = []
-				while True: 
-					notes = input("")
-					if notes == "*END":
-						break
-					newContent1.append(notes)
-				newContent = ""
-				for line in newContent1: 
-					newContent += line + "\n"
-				apiCalls.createNote(newNoteName,newContent)
-				printColor("Would you like to add any tags? (Seperate tags with commas. If none or you wish to add later, enter 'none'.)",getConfig(2),"")
-				newTags = input(": ")
-				newTagList = newTags.split(",")
-				if(newTags == 'none'):
-					lineBreak(columns, getConfig(4))
-					printColor("Note Created!",getConfig(2))
-					time.sleep(2)
-					clearConsole()
-					printAppUse()
-					state = 1
-				else:
-					apiCalls.addTag(newNoteName,newTagList)
-					lineBreak(columns,getConfig(4))
-					printColor("Note Created!", getConfig(2))
-					time.sleep(2)
-					clearConsole()
-					printAppUse()
-					state = 1
-				
-	#editNote
-			case 12:
-				lineBreak(columns, getConfig(4))
-				printColor("Editing a note...",getConfig(2))
-				lineBreak(columns, getConfig(4))
-				printColor("0. Go back",getConfig(2),"")
-				printColor("1. Add Content",getConfig(2),"")
-				printColor("2. Edit Content (temporarily not working. Coming in v0.02)",getConfig(2),"")
-				printColor("3. Add tags to note",getConfig(2),"")
-				printColor("4. Delete tags from note", getConfig(2))
-				
-		#addContent
-			case 121:
-				lineBreak(columns, getConfig(4))
-				printColor("Adding content...",getConfig(2))
-				lineBreak(columns, getConfig(4))
-				printColor("What is the name of the note you wish to add to?",getConfig(2),"")
-				contentName = input(": ")
-				printColor("Write the content to wish to add.",getConfig(2),"")
-				printColor("*END: Stop typing to file.",getConfig(2),"")
-				printColor("return key: start a new line.",getConfig(2),"")
-				addContent1 = []
-				while True: 
-					notes = input("")
-					if notes == "*END":
-						break
-					addContent1.append(notes)
-				for line in addContent1: 
-					addContent += line + "\n"
-				apiCalls.addContent(contentName,addContent)
-				lineBreak(columns, getConfig(4))
-				printColor("Added content!",getConfig(2))
-				time.sleep(2)
-				clearConsole()
-				printAppUse()
-				state = 1
-				
-		#editContent limited/cut temporarily for time
-			case 122:
-				lineBreak(columns, getConfig(4))
-				printColor("Coming in v0.02.",getConfig(2))
-				lineBreak(columns, getConfig(4))
-				time.sleep(2)
-				clearConsole()
-				printAppUse()
-				state = 1
-
-		#addTag
-			case 123:
-				lineBreak(columns, getConfig(4))
-				printColor("Adding tag...",getConfig(2),"")
-				lineBreak(columns, getConfig(4))
-				printColor("What is the name of the note you wish to edit?",getConfig(2),"")
-				addTagName = input(": ")
-				printColor("What is the tags that you wish to add? (List with commas inbetween)", getConfig(2),"")
-				addTags = input(": ")
```

Comment:
why did you delete these?

Reply:
These blocks were removed because a separate functionality for managing tags (adding, deleting, renaming, etc.) was introduced through a new state case 16 and the printTagMenu() function.The printTagMenu() function was introduced, which displays a menu for tag management operations. This menu allows users to add, delete, list, search, and rename tags without being tied to a specific note.(not much change actually)